deployments/container/Makefile

```diff
@@ -27,7 +27,8 @@ ifeq ($(IMAGE_NAME),)
 IMAGE_NAME := $(REGISTRY)/$(DRIVER_NAME)
 endif
 
-IMAGE_VERSION := $(VERSION)
+# Here we would like to use the v prefix.
+IMAGE_VERSION := $(vVERSION)
```

Comment:
Kevin asked via Slack

>  the container image should still have the v

this change ensures we use the v prefix here.

Reply:
It's not quite as simple as that ... 
the SHA-tagged images shouldn't have a `v` prefix.

Reply:
> SHA-tagged images shouldn't have a v prefix.

Okay. That means we must not set `IMAGE_VERSION := $(vVERSION)`.

We could do 

```
IMAGE_VERSION := $(git rev-parse --short=8)
```

instead in case of CI. But my intuition is that we don't like this change.

We have a new `package-helm-charts.sh` in here that always wants a version w/o `v` prefix:

https://github.com/NVIDIA/k8s-dra-driver-gpu/blob/8d34f977b7bdd7b7cad8c3fd0d53229b1676ca0f/hack/package-helm-charts.sh#L25

So, I will revert the `IMAGE_VERSION` change and amend `package-helm-charts.sh` to conditionally strip a `v` prefix from `VERSION`.

Reply:
Done the above.